app/routing/Versions.scala

```diff
@@ -67,12 +67,17 @@ case object Version4 extends Version {
   val name = "4.0"
 }
 
+case object Version5 extends Version {
```

Comment:
You need to update `VersionSpec.scala` as part of this PR. You can take a look at this PR for reference: https://github.com/hmrc/individual-losses-api/pull/457/files

Reply:
I don't think a change in VersionSpec is required as it's not testing the code for each version individually. However, I have updated to test it for the latest version for clarity.